tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
in `@A` rule `AA` is on
in `@C` rule `AA` is off

`@A - @C` should still has `AA` on, as we didn't extract any set that contains this rule